pyartcd/config.example.toml

```diff
@@ -12,12 +12,32 @@ smtp_server = "smtp.corp.redhat.com"
 from = "aos-art-automation@redhat.com"
 reply_to = "aos-team-art@redhat.com"
 cc = ["aos-team-art@redhat.com"]
-prepare_release_notification_recipients_ocp4=["multi-arch-zstream-testing@redhat.com", "aos-qe@redhat.com"]
+prepare_release_notification_recipients_ocp4=["multi-arch-zstream-testing@redhat.com", "aos-qe@redhat.com", "mp-entitlement-qe@redhat.com"]
 prepare_release_notification_recipients_ocp3=["wzheng@redhat.com", "gpei@redhat.com", "aos-qe@redhat.com", "mp-entitlement-qe@redhat.com", "openshift-lp-test@redhat.com", "pprakash@redhat.com"]
 promote_image_list_recipients = ["openshift-ccs@redhat.com"]
 
 [jira]
 url = "https://issues.redhat.com"
 [jira.templates]
-ocp4 = "OCPPLAN-4756"
-ocp3 = "OCPPLAN-1373"
+ocp4 = "ART-4223"
+ocp3 = "ART-4234"
+
+[message_broker.signing_stage]
+host_and_ports = [['umb-broker03.stage.api.redhat.com', 61612], ['umb-broker04.stage.api.redhat.com', 61612]]
+# cert_file = "ssl/nonprod-openshift-art-bot.crt"
+# key_file = "ssl/nonprod-openshift-art-bot.key"
+
+[signing_stage]
+message_broker = "signing_stage"
+send_destination = '/topic/VirtualTopic.eng.art.artifact.sign'
+recv_destination = '/queue/Consumer.nonprod-openshift-art-bot.artcd-stage.VirtualTopic.eng.robosignatory.art.sign'
+
+[message_broker.signing]
+host_and_ports = [['umb-broker03.api.redhat.com', 61612], ['umb-broker04.api.redhat.com', 61612]]
```

Comment:
Think we should use the user facing connection urls as defined here: https://source.redhat.com/groups/public/enterprise-services-platform/it_platform_wiki/umb_client_guide in section _UMB Connection Urls_.
That'd mean we do not have an outage if a host needs to get rotated.